tests/shared/src/test/scala/org/http4s/LiteralSyntaxMacrosSuite.scala

```diff
@@ -18,10 +18,20 @@ package org.http4s
 
 import org.http4s.implicits._
 
+import scala.annotation.nowarn
+
 class LiteralSyntaxMacrosSuite extends Http4sSuite {
   test("'uri' macro works for valid input") {
     assertEquals(uri"a.b.c", Uri.fromString("a.b.c").toOption.get)
   }
+  test("'uri' macro works with variable named 'org'") {
```

Comment:
would this now crash with `_root_` ?

Reply:
Yes, it starts failing with the error `value org is not a member of String`  (I added `implicit val _https_ = "blerf"`)

I think `_root_` as a variable name is a lot less common than `org` though?